src/Forms/ConfirmedPasswordField.php

```diff
@@ -25,34 +27,33 @@ class ConfirmedPasswordField extends FormField
 
     /**
      * Minimum character length of the password.
-     *
-     * @var int
      */
-    public $minLength = null;
+    public int $minLength = 0;
 
     /**
      * Maximum character length of the password.
-     *
-     * @var int
+     * 0 means no maximum length.
      */
-    public $maxLength = null;
+    public int $maxLength = 0;
 
     /**
-     * Enforces at least one digit and one alphanumeric
-     * character (in addition to {$minLength} and {$maxLength}
-     *
-     * @var boolean
+     * Enforces password strength validation based on entropy.
+     * See setMinPasswordStrength()
      */
-    public $requireStrongPassword = false;
+    public bool $requireStrongPassword = false;
 
     /**
      * Allow empty fields when entering the password for the first time
      * If this is set to true then a random password may be generated if the field is empty
      * depending on the value of $ConfirmedPasswordField::generateRandomPasswordOnEmtpy
-     *
-     * @var boolean
      */
-    public $canBeEmpty = false;
+    public bool $canBeEmpty = false;
+
+    /**
+     * Minimum password strength if requireStrongPassword is true
+     * See https://symfony.com/doc/current/reference/constraints/PasswordStrength.html#minscore
+     */
+    private int $minPasswordStrength = PasswordStrength::STRENGTH_STRONG;
```

Comment:
What does STRENGTH_STRONG mean?

Does it different much from what "strong" used to mean in CMS 5?

Reply:
It is a name to match a (fairly arbitrary from what I can tell) level of entropy. It's less strong than `STRENGTH_VERY_STRONG` and stronger than `STRENGTH_MEDIUM`.

Reply:
The link to the relevant symfony docs is right there, I don't have any more information than symfony provides about how symfony works.